
### Developing SNPs in multiple species of pangolins for population differentiation

The goal of this project is to build a reproducible pipeline that takes whole genome sequence data from pangolins and identifies single nucleotide polymorphisms (SNPs) for use in genetic assignment tests.


#### Objectives
Create a well documented and reproducible pipeline that:
1. Identifies SNPs across species and populations
2. Designs multiplex primers for SNPs of interest

#### Repository Structure

[**data**](https://github.com/fish546-2018/hj-pangolin/tree/master/data)  
README and files containing information about data files. Raw data files are too big to store on Github.

[**tutorials**](https://github.com/fish546-2018/hj-pangolin/tree/master/tutorials)  
Jupyter and R notebooks from tutorials in class.   
  - BLAST tutorial

[**notebooks**](https://github.com/fish546-2018/hj-pangolin/tree/master/notebooks)  
Jupyter notebooks used for analyses.

[**scripts**](https://github.com/fish546-2018/hj-pangolin/tree/master/scripts)  
Bash and python scripts used to run analyses on Mox.


#### Project Timeline

**Week 4:** Set up project directory and organization for running analyses on Mox.

**Week 5:** Run ```FASTQC``` on raw sequences files using GNU parallel to learn how to split up commands

![pangolin](https://farm5.staticflickr.com/4421/35769436014_4e879dcea6_z_d.jpg)  Adam Tusk / CC BY 2.0
